src/features/lightspeed/api.ts

```diff
@@ -101,32 +101,32 @@ export class LightSpeedAPI {
       if (err && "response" in err) {
         if (err?.response?.status === 401) {
           vscode.window.showErrorMessage(
-            "User not authorized to access Ansible Lightspeed service."
+            "User not authorized to access Ansible Lightspeed."
           );
         } else if (err?.response?.status === 429) {
           vscode.window.showErrorMessage(
-            "Too many requests to the Ansible Lightspeed service. Please try again after sometime."
+            "Too many requests to the Ansible Lightspeed. Please try again after sometime."
```

Comment:
With service removed, we need to remove "the". While we're on this line, "sometime" should be "some time".